src/API.CRUD/Helpers/JsonHelpers.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Serialization;
+
+namespace API.CRUD
+{
+    public static class JsonHelpers
+    {
+        private static readonly Lazy<JsonSerializerSettings> LazyJsonSerializerSettings = new Lazy<JsonSerializerSettings>(() =>

```

Comment:
Does this need to be _lazy_? Seems like something that going to be needed pretty much immediately (as opposed to a service used somewhere/sometimes, so we don't want to init it straight away)

Reply:
Remove lazy.

Reply:
🚮